src/InternetAddress.php

```diff
@@ -75,7 +80,7 @@ public function getAddressBytes(): string
         return $this->binaryAddress;
     }
 
-    public function getType(): InternetAddressType
+    public function getVersion(): InternetAddressType
```

Comment:
Should be `InternetAddressVersion` then?

Reply:
I agree, yeah.